src/gui/gui.h

```diff
@@ -97,6 +97,23 @@ class DissolveWindow : public QMainWindow
     // List of ReferencePoints currently loaded
     std::vector<ReferencePoint> referencePoints_;
 
+    private:
+    QList<QAction *> recentFileActionList;
+    const int recentFileNo;
```

Comment:
For us, private member variables always have a suffixed underscore:
```suggestion
    QList<QAction *> recentFileActionList_;
    const int recentFileNo_;
```